### ISSUE TITLE
Hyperparameters for RVL-CDIP 

### ISSUE THREAD

**Issue by saifullah3396** [microsoft/unilm#635]
Hi, 

I am trying to reproduce the results of LayoutLMV2 (Base) on RVL-CDIP. I have used the IIT-CDIP OCR so there's no problem related to that. I am able to get accuracy up to 94.0%, therefore I believe the results are reproducible but I would really like to reach the mentioned accuracy of 95.25%. Would it be possible for you guys to share the exact hyperparameter setting that you used for fine-tuning the network on RVL-CDIP?

Thanks
